packages/react-query/src/bin/cli.ts

```diff
@@ -34,4 +37,10 @@ program
   )
   .action(fixAction)
 
-program.parse(process.argv)
+try {
+  program.parse(process.argv)
+} catch (error) {
+  if (error instanceof Error) {
+    logger.error(error.message)
+  }
+}
```

Comment:
All throw Errors occurring during the CLI process are handled by the designated error boundary